types/module/server/manager.go

```diff
@@ -115,6 +128,22 @@ func (mm *Manager) RegisterModules(modules []module.Module) error {
 	return nil
 }
 
+// WeightedOperations returns all the modules' weighted operations of an application
+func (mm *Manager) WeightedOperations(state sdkmodule.SimulationState, modules []sdkmodule.AppModuleSimulation) []simulation.WeightedOperation {
+	wOps := make([]simulation.WeightedOperation, 0, len(modules))
```

Comment:
Yeah, I'm getting a lint issue.

Reply:
well in this case, let's keep using `make` but with a capacity that makes sense ie `len(modules) + len(mm.weightedOperationsHandlers)` and that will avoid any reallocation.

Reply:
unresolving this as this hasn't been addressed

Reply:
We cannot use `len(modules) + len(mm.weightedOperationsHandlers)` . Each module returns [`[]simulation.WeightedOperation` ](https://github.com/regen-network/regen-ledger/pull/251/files#diff-a3144c816ac58cb8fbf744d53aa6cc93d3601da586fb23a432db1909b3c4eff8R134-R142).

Reply:
oh I see, they might actually be more elements in `wOps` than `len(modules) + len(mm.weightedOperationsHandlers)` but still, pre-allocating would avoid reallocation until we reach `len(modules) + len(mm.weightedOperationsHandlers)` (whereas using `var wOps []simulation.WeightedOperation`, it will realloc from the very first `append`)